packages/playout-gateway/src/tsrHandler.ts

```diff
@@ -273,6 +280,28 @@ export class TSRHandler {
 		this._triggerUpdateDevices()
 		this.logger.debug('tsr init done')
 	}
+
+	private loadSubdeviceConfigurations(): { [deviceType: string]: Record<string, any> } {
```

Comment:
Perhaps `{ [deviceType: string]: Record<string, any> }` should be extracted to a type, since it is being used in multiple places.